src/Common/src/Interop/Unix/libcrypto/Interop.Initialization.cs

```diff
@@ -21,8 +18,15 @@ static libcrypto()
                 throw new CryptographicException();
             }
 
+            // Load the SHA-2 hash algorithms, and anything else not in the default
+            // support set.
+            OPENSSL_add_all_algorithms_conf();
+
             // Ensure that the error message table is loaded.
-            Interop.libcrypto.ERR_load_crypto_strings();
+            ERR_load_crypto_strings();
         }
```

Comment:
OpenSSL normal API/library errors are a stack.  The chain validation errors are [X509_STORE_CTX_get_error](https://www.openssl.org/docs/crypto/X509_STORE_CTX_get_error.html), which talks a lot about *the* error.  But I'm certainly open to finding out that more than one error is possible, and from more than one certificate in the chain.